WordPressAuthenticator/Unified Auth/GoogleAuthenticator.swift

```diff
@@ -120,11 +120,36 @@ class GoogleAuthenticator: NSObject {
     ///                  The values are updated during the Google process,
     ///                  and returned to the calling view controller via delegate methods.
     ///   - authType: Indicates the type of authentication (login or signup)
-    func showFrom(viewController: UIViewController, loginFields: LoginFields, for authType: GoogleAuthType = .login) {
+    func showFrom(
+        viewController: UIViewController,
+        loginFields: LoginFields,
+        for authType: GoogleAuthType = .login
+    ) {
+        // The fact that we set `loginFields`, then reset its `meta.socialService` property doesn't
+        // seem ideal...
         self.loginFields = loginFields
         self.loginFields.meta.socialService = SocialServiceName.google
         self.authType = authType
-        requestAuthorization(from: viewController)
+
+        guard authConfig.googleLoginWithoutSDK else {
+            // Use method that depends on SDK
+            requestAuthorization(from: viewController)
+            return
+        }
+
+        Task {
```

Comment:
Similar to https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/743/files#r1110446829, it's probably safer to run this Task in the main actor (i.e. adding a `@MainActor` annotation: `Task { @MainActor in `), so that the `didSignIn` function below are guaranteed to be called from the main thread.

Reply:
Good point. 

Addressed in 6e77d0aa. Thanks!